docs/ui/components/Step/Step.tsx

```diff
@@ -0,0 +1,66 @@
+import { css } from '@emotion/react';
+import { spacing, theme, typography } from '@expo/styleguide';
+import { PropsWithChildren } from 'react';
+
+type Props = PropsWithChildren<{
+  label: string;
+  connected?: boolean;
+}>;
+
+export const Step = ({ children, label, connected }: Props) => {
+  return (
+    <div css={stepWrapperStyle}>
+      <div>
+        <div css={stepLabelStyle}>{label}</div>
```

Comment:
Can we use `<HEADLINE>` here instead of defining the text styles manually with the `stepLabelStyle`?